modules/structs/repo_branch.go

```diff
@@ -136,6 +136,7 @@ type UpdateBranchProtectionPriories struct {
 
 type MergeUpstreamRequest struct {
 	Branch string `json:"branch"`
+	FfOnly *bool  `json:"ff_only,omitempty"`
```

Comment:
```suggestion
	FfOnly bool   `json:"ff_only"`
```

I think it doesn't need to use pointer here. And CI failure seems related.